index.bs

```diff
@@ -164,9 +164,13 @@ dictionary which describes modifications to the sanitze operation.
 :: The <dfn>element drop list</dfn> is a sequence of strings with elements
    that the sanitizer should remove from the input, including its children.
 : allowAttributes
-:: TODO: <dfn>attribute allow list</dfn>
+:: The <dfn>attribute allow list</dfn> is a map of attributes names a list of
+   elements names, describing which attributes should be allowed on which
+   elements. The special "name" `"*"` denotes all elements.
```

Comment:
Nit: s/elements/attributes (also occurs on line 173)

Do we really want to offer that kind of footgun?

Reply:
I think I meant elements, though.

The (main) idea is that one allows/blocks element/attribute combinations. (Like: script.src) But one might also block attributes absolutely (like: *.src). The main reason why I included that last option is because some existing santizers allow that, and I don't want to force people to re-write all their logic when migrating to sanitizer API. It might also make a lot of sense for some "generic" attributes like event handlers. (Although those would be blocked (and hence non-allowable) by the default config anyhow.)

The notation here is: attr -> [ elements ]. So:

- "src": [ "script" ]
- "onclick": [ "*" ]


With that said: I'm not seeing a footgun. (I do see complexity, which might not be warranted. But I'm seeing anything obvious "sharp edge" in which the user could harm themselves here.)

At the very least, I need to clarify the intent, though. :)

Reply:
Maybe done. :-)

I've re-written this to be clear(er), hopefully, but have not changed the substance of it. Does this make sense?